pkg/stanza/operator/helper/writer.go

```diff
@@ -44,6 +46,7 @@ type WriterOperator struct {
 	BasicOperator
 	OutputIDs       []string
 	OutputOperators []operator.Operator
+	ZapLogger       *zap.Logger
```

Comment:
I don't think you need to make any changes to the helper package. The `BasicOperator` already has a a `TelemetrySettings` and a `Logger()` method.

Reply:
I see, Thanks. Removed code in helper package.